leaderboardFeature/src/main/java/com/panhuk/leaderboardfeature/utils/Image.kt

```diff
@@ -0,0 +1,20 @@
+package com.panhuk.leaderboardfeature.utils
+
+import com.panhuk.leaderboardfeature.R
+import kotlin.random.Random
+
+fun getDrawable(index: Int = Random.nextInt(11)): Int {
```

Comment:
list can be created only once, on top level, for example

private val list = //

fun getDrawableRes() : Int = list[Random.nextInt(list.size-1)]

Reply:
Wow, very cool